src/Pages/EventManager/EventManagerModal.js

```diff
@@ -51,7 +54,12 @@ function EventManagerModal(props) {
         addon: 'Event Date*',
         type: 'date',
         defaultValue: props.eventDate ? props.eventDate.slice(0, 10) : '',
-        handleChange: e => setEventDate(e.target.value)
+        handleChange: e => {
+          setEventDate(e.target.value);
+          if(eventDate === undefined) {
```

Comment:
why do we check this?

Reply:
I was attempting to create an alert whenever the user enters an invalid date when creating an event so I left that code in there for testing purposes, but I was never able to successfully resolve that issue.